build/makelib/golang.mk

```diff
@@ -183,6 +187,14 @@ $(GO_TOOLS_DIR)/glide:
 	@mv $(GO_TOOLS_DIR)/$(GOHOSTOS)-$(GOHOSTARCH)/glide $(GO_TOOLS_DIR)/glide
 	@rm -r $(GO_TOOLS_DIR)/$(GOHOSTOS)-$(GOHOSTARCH)
 
+$(GO_TOOLS_DIR)/golint:
```

Comment:
Good point. This will check if the golint is already built. If it is, it will skip it